pkg/apis/pipeline/v1/when_validation.go

```diff
@@ -20,6 +20,7 @@ import (
 	"fmt"
 	"strings"
 
+	_ "github.com/google/cel-go/cel"
```

Comment:
Maybe add a todo comment here?

Reply:
Thanks! Added a todo linking to the tracking issue